src/Catty UITests/ScriptCollectionView/BrickCellPickerViewerTest.swift

```diff
@@ -76,7 +76,7 @@ class BrickCellPickerViewerTests: XCTestCase {
         }
     }
 
-    func testChangeVariableDone() {
+    func testChangeVariable() {
```

Comment:
Could you please change the name to something like "testChangeVariableDoneAndCancel"? I think this would be more intuitive.